### PR TITLE
feat(stock_entry): stock Entry dashboard added for quality inspection reference

### PR DESCRIPTION
**Pull request by vishdha**
TASK ID: [asana](https://app.asana.com/0/1194641031921089/1193627216589857)

Problem Description:
As User I want to have a Quality Inspection Reference in stock entry.
Right now there is no way to check from the stock entry that which Quality Inspection made against the Stock Entry. 

Solution:
Material-Transfer-for-Manufacture-MAT-STE-2020-00308

Steps to Reproduce:
1. Create stock entry
2. Create Quality Inspection against stock entry
3. Submit Quality Inspection will update Quality Inspection name in items child table.
4. Refresh stock entry
5. Moved Create Material Request button in Create drop-down

### CODE REVIEW

File: erpnext/stock/doctype/stock_entry/stock_entry_dashboard.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from __future__ import unicode_literals
+from frappe import _
+
+def get_data():
+	return {
+		'fieldname': 'stock_entry',
+        'non_standard_fieldnames': {
```
Comment (Alchez) [inline erpnext/stock/doctype/stock_entry/stock_entry_dashboard.py]: Change all spaces in this file to tabs

File: erpnext/stock/doctype/stock_entry/stock_entry_dashboard.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+			'Qulaity Inspection': 'reference_name'
```
Comment (Alchez) [inline erpnext/stock/doctype/stock_entry/stock_entry_dashboard.py]:
```suggestion
			'Quality Inspection': 'reference_name'
```

File: erpnext/stock/doctype/stock_entry/stock_entry_dashboard.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+		},
+		'internal_links': {
+			'Quality Inspection': ['items', 'quality_inspection']
+		},
+        'transactions': [
+			{
+				'label': _('Reference'),
+				'items': ['Quality Inspection']
+			}
+		]
```
Comment (Alchez) [inline erpnext/stock/doctype/stock_entry/stock_entry_dashboard.py]: If we're creating this anyway, should we add other links as well (for example, Work Order)?

**Review by Alchez — CHANGES_REQUESTED** [review]

File: erpnext/stock/doctype/stock_entry/stock_entry_dashboard.py (hunk 3 above)
Comment (vishdha) [reply erpnext/stock/doctype/stock_entry/stock_entry_dashboard.py]: Material Request and Purchase Receipt link added.

Comment (Alchez) [reply erpnext/stock/doctype/stock_entry/stock_entry_dashboard.py]: The following doctypes are linked to Stock Entry (or found in the items table):

```
Work Order
Purchase Order
Delivery Note
Sales Invoice
BOM
Warehouse
Address
Supplier
Customer
Project
Job Card
Pick List
Batch
Package Tag
```

Talk to Arundhati and Swati, and see if we should pull any of these?

File: erpnext/stock/doctype/stock_entry/stock_entry.js [hunk 4]
```diff
@@ -149,7 +153,7 @@ frappe.ui.form.on('Stock Entry', {
 					});
 					frappe.set_route('Form', 'Material Request', mr.name);
 				});
-			});
+			}, __("Create"));
```
Comment (Alchez) [inline erpnext/stock/doctype/stock_entry/stock_entry.js:156]: I think we should start removing all the "Create" buttons, and put that functionality in the + button, the "Create" should be used only when it needs to happen conditionally.

Comment (neilLasrado) [reply erpnext/stock/doctype/stock_entry/stock_entry.js:156]: @Alchez yeah makes sense. Let's make a separate task for this & handle it in a separate PR

**Review by neilLasrado — APPROVED** [review]

File: erpnext/stock/doctype/stock_entry/stock_entry_dashboard.py (hunk 3 above)
Comment (neilLasrado) [reply erpnext/stock/doctype/stock_entry/stock_entry_dashboard.py]: Let's make separate tasks for these

**Pull request merged**